### PR TITLE
OKRs - 2019 Q4 - Project Operations

### PR DESCRIPTION
**Pull request by momack2**
Here's a first draft of project operations Q4 OKRs! would love feedback from @parkan @autonome @alanshaw @achingbrain @hugomrdias @lidel @daviddias etc

Ref https://github.com/ipfs/team-mgmt/issues/1033

### CODE REVIEW

**Comment by autonome** [comment]
Yesterday the Browser SIG narrowed Q4 priorities to:

* Brave & Opera support work
* Testing, both core and GUIs
* Async/await refactor

Woohoo, totally aligned with the draft OKRs 🎉

Differences:

* We're going to carry forward the Q3 testing KR, to continue work on reducing the time it takes for tests to run, and reduce the complexity and overhead of the test infra.
* We'll expand the GUI testing KR to include test automation for Companion

**Comment by parkan** [comment]
this looks good to me currently, thanks @momack2

will PR with more details for metrics next week (currently in a Dr waiting room with back spasms, in addition to technically being afk for the wedding)

File: OKR/PROJECT.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2019 Q4
+- **O: IPFS Project prepared for 2020 growth** 
+  - Onboard 4+ new contributors with skills around Testing infra, Gateways support, go-ipfs swe, and docs content strategy
+  - 2020 planning identifies our next area of focus and starts building toward it
```
Comment (daviddias) [inline OKR/PROJECT.md]: Wondering if could be at some point "<Visionary Goal part Deux>"

Comment (momack2) [reply OKR/PROJECT.md]: Good idea!
```suggestion
  - 2020 planning identifies our next area of focus and updates our roadmap by Nov 16
```

**Comment by momack2** [comment]
I'm merging this branch - @parkan, @autonome - please make updates directly in the sheet: https://docs.google.com/spreadsheets/d/1VeyiLvBdX_PrP394kU_lwkQZxfNwqMVX1f7K4ursSPM/edit#gid=1562851442

**Pull request merged**